src/build/dotnet/steps/net-core.steps.yaml

```diff
@@ -0,0 +1,73 @@
+# Steps to build .NET Core Projects
+parameters:
+  - name: projects
+    default: '**/*.csproj'
+  - name: tests
+    default: '**/*.Tests.csproj'
+  - name: shouldRunTests
+    default: true
+  - name: applyVersioning
+    default: true
+  - name: artifactName
+    default: '$(Build.DefinitionName)'
+
+steps:
+  - ${{ if eq(parameters.applyVersioning, true) }}:
+    - task: UpdateDotNetVersions@12
+      displayName: .NET Versioning
+      inputs:
+        releaseBranchNames: 'master,main'
```

Comment:
releaseBranchNames is parameterised in the nuget jovs - should we do here also?

Reply:
Have added here